deployment/ansible/roles/district-builder.app-server/README.md

```diff
@@ -0,0 +1,31 @@
+district-builder.app-server
+=========
+
+Ansible role to configure a DistrictBuilder application server.
+
+Role Variables
+--------------
+
+**`districtbuilder_app_home`**: Directory where configuration files and data will be loaded. Default: `/opt/district-builder`
+
+**`districtbuilder_image_version`**: Version of DistrictBuilder to run. Default: `latest`
+
+**`districtbuilder_user_config_file`**: Use a user-provided `config.xml` file to generate settings. Default: `true`. ***Note: If this is `true`, place the config file in the `files/` folder.
+
+**`districtbuilder_web_app_password`**: Django user password. Default: `districtbuilderdistrictbuilder`
```

Comment:
Could we make passwords required and not just have some really insecure default? I think we intentionally left them blank in `.env.sample` up until now so that people would have to fill something in.

Reply:
This file is actually managed by TF now, so there are technically no more defaults. This file should just be deleted.